### PR TITLE
Better and more colorful boot verbose

### PR DESCRIPTION
**Pull request by Doges**
image

# What was done so far
- [x] Make APIs a CommandRequirement and log when a command doesn't load because one is missing
- [x] Make environment variables a CommandRequirement and log when a command doesn't load because one is missing
- [x] Make environment variables a requirement (how?) for APIs and log when a command doesn't load because one is missing
- [x] Let `canLoad` return a String and use it as the error text
- [x] Check for all three required environment variables for Lavalink instead of just `LAVALINK_HOST`
- [x] Change all color codes to be between 30 and 37 (papertrail only supports those)# TO-DOs

### CODE REVIEW

**Comment by pedrofracassi** [comment]
Cool I lost all of my credits on this one :haaa

**Comment by Doges** [comment]
> Cool I lost all of my credits on this one :haaa

stfu

**Comment by pedrofracassi** [comment]
# New TO-DOs
- [ ] Change all color codes to be between 30 and 37 (papertrail only supports those)

File: index.js [hunk 1]
```diff
@@ -11,4 +11,4 @@ const CLIENT_OPTIONS = {
 
 const Switchblade = require('./src/Switchblade.js')
 const client = new Switchblade(CLIENT_OPTIONS)
-client.login().then(() => client.log('Logged in successfully!', 'Discord')).catch(e => client.logError(e))
+client.login().then(() => client.log('[92mLogged in successfully!', 'Discord')).catch(e => client.logError(e))
```
Comment (pedrofracassi) [inline index.js]:
```suggestion
client.login().then(() => client.log('[32mLogged in successfully!', 'Discord')).catch(e => client.logError(e))
```

**Comment by pedrofracassi** [comment]
@Doges what else do we have to do?

**Comment by Doges** [comment]
> @Doges what else do we have to do?

nothing i think

**Comment by Doges** [comment]
O
I WANT SOMETHING

**Comment by Doges** [comment]
RFR

**Comment by bolsomito** [comment]
is it really rfr?

**Comment by Doges** [comment]
> is it really rfr?

i said rfr so, yes

File: src/listeners/MainListener.js [hunk 2]
```diff
@@ -22,9 +23,9 @@ module.exports = class MainListener extends EventListener {
         user: this.user.id,
         shards: 1
       })
-      this.log('Player manager connection established!', 'Music')
+      this.log('[32mLavalink connection established!', 'Music')
     } else {
-      this.log('Player manager connection didn\'t establish!', 'Music')
+      this.log(`[31mLavalink connection didn't establish - Required environment variable(s) (${lavalinkRequiredVariables.filter(variable => !process.env[variable]).join(', ')}) not set.`, 'Music')
```
Comment (Lireoy) [inline src/listeners/MainListener.js]: Probable improvement here could be:
`Failed to establish Lavalink connection`

**Comment by pedrofracassi** [comment]
Seems to be good to go!

**Pull request merged**